adduser.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+#############for test only################
```

Comment:
For security reasons I wouldn't add this to an actual file, but instead for the time being update the README.md file showcasing how to write this into a file and execute it (for example could go to a new subsection before [the testing one](https://github.com/Kartones/flask-calendar#testing)).